### PR TITLE
Allow concurrent Jenkins runs - Closes #300

### LINKED ISSUE #300: Allow concurrent Jenkins runs
### Expected behaviour
Runs can be done concurrently
### Actual behaviour
Runs are serial

### PR DESCRIPTION
**Pull request by Nazgolze**
- Update to Jenkins-files to allow concurrent builds.
 - Improve error output.

Closes #300

### CODE REVIEW

**Comment by Nazgolze** [comment]
no, this did not pass. looking into it

File: Jenkinsfile [hunk 1]
```diff
@@ -49,34 +58,53 @@ node('lisk-explorer-01'){
...
+        cp .config.json config_$N.json
+        # change core port
+        sed -i -r -e "s/^(.*ort\\":) 4000,/\\1 400$N,/" config_$N.json
+        # disable redis
+        sed -i -r -e "s/^(\\s*\\"cacheEnabled\\":) true/\\1 false/" config_$N.json
+        # change postgres databse
+        sed -i -r -e "s/^(\\s*\\"database\\": \\"lisk_test)\\",/\\1_$N\\",/" config_$N.json
+        cp etc/pm2-lisk.json etc/pm2-lisk_$N.json
+        sed -i -r -e "s/config.json/config_$N.json/" etc/pm2-lisk_$N.json
+        sed -i -r -e "s/(lisk.app)/\\1_$N/" etc/pm2-lisk_$N.json
```
Comment (Isabello) [inline Jenkinsfile]: my recommendation is to keep a preconfigured config.json in the test folder of explorer that can be copied into place, removing this messy logic.

File: Jenkinsfile [hunk 2]
```diff
@@ -118,5 +147,33 @@ node('lisk-explorer-01'){
...
+    N=${EXECUTOR_NUMBER:-0}
+    ( cd ~/lisk-Linux-x86_64 && bash lisk.sh stop_node -p etc/pm2-lisk_$N.json ) || true
+    pkill -f "Xvfb :1$N" -9 || true
+    pkill -f "webpack.*808$N" -9 || true
+    '''
+    dir('node_modules') {
+      deleteDir()
+    }
+
+    def pr_branch = ''
```
Comment (Isabello) [inline Jenkinsfile]: this function def should be moved to the top of the file

File: Jenkinsfile.mock [hunk 3]
```diff
@@ -53,22 +62,28 @@ node('lisk-explorer-01'){
...
       try {
+
         sh '''
         N=${EXECUTOR_NUMBER:-0}
         cd lisk-mock
         npm install
         PORT=808$N npm run dev > results.txt &
         '''
+
+        /*sh '''#!/bin/bash
```
Comment (Isabello) [inline Jenkinsfile.mock]: is this commented out ?

File: Jenkinsfile.mock [hunk 4]
```diff
@@ -124,5 +140,33 @@ node('lisk-explorer-01'){
...
+    N=${EXECUTOR_NUMBER:-0}
+    ( cd ~/lisk-Linux-x86_64 && bash lisk.sh stop_node -p etc/pm2-lisk_$N.json ) || true
+    pkill -f "Xvfb :1$N" -9 || true
+    pkill -f "webpack.*808$N" -9 || true
+    '''
+    dir('node_modules') {
+      deleteDir()
+    }
+
+    def pr_branch = ''
```
Comment (Isabello) [inline Jenkinsfile.mock]: same as other comment about this.

**Comment by Nazgolze** [comment]
This is quite stale. Closing this and opening a new one

**Pull request closed without merging**